.github/workflows/ci_cron_weekly.yml

```diff
@@ -113,17 +113,16 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: uraimo/run-on-arch-action@v2.1.1
+      - uses: uraimo/run-on-arch-action@v2
         name: Run tests
         id: build
         with:
           arch: ${{ matrix.arch }}
-          distro: bullseye
+          distro: ubuntu22.04
```

Comment:
Wasn't this changed to `ubuntu_latest` already?

Reply:
Huh! Did I accidentally revert it in rebase?

Reply:
Fixed. Good catch. Thanks! Please re-review.